docs/spelling_wordlist

```diff
@@ -2,6 +2,7 @@
 accessor
 accessors
 Aceh
+ACos
```

Comment:
I don't think these should be needed. The functions should always be in backticks prevents them from being spell checked.

Reply:
OK. I removed all the function names I added.